master/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -1,45 +1,69 @@
-use the [Calico only manifest](../hosted) instead, as it does not deploy its
-own etcd.
+## Installation
+
+You can easily create a cluster compatible with these manifests by following [the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
+
+Calico installations require kubeadm initialized clusters to have set a `--pod-network-cidr`
+which should match the value set by `CALICO_IPV4POOL_CIDR` in the manifest:
 
-You can easily create a cluster compatible with this manifest by following [the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
+```yaml
+- name: CALICO_IPV4POOL_CIDR
+  value: "192.168.0.0/16"
+```
 
+This value can be changed to any CIDR that does not overlap the cluster's service CIDR.
+The manifests are currently setup to use the CIDR `192.168.0.0/16` so you would initialize
+the cluster with the following command:
 
-#### Installation
+```shell
+kubeadm init --pod-network-cidr=192.168.0.0/16
+```
+
+### etcd datastore
+
+Users who have deployed their own etcd cluster outside of kubeadm should
+use the [Calico only manifest](../hosted) instead, as it does not deploy its
+own etcd.
 
 To install this Calico and a single node etcd on a run the following command
 depending on your kubeadm / kubernetes version:
 
-For Kubeadm stable with Kubernetes version >= v1.6.0:
+For kubeadm stable with Kubernetes version >= v1.6.0:
 
-```
+```shell
 kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubeadm/1.6/calico.yaml
 ```
 
 >[Click here to view the above yaml directly.](1.6/calico.yaml)
 
-For Kubeadm 1.5 with Kubernetes version v1.5.x:
+### Kubernetes datastore
 
-```
-kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubeadm/1.5/calico.yaml
+To install Calico configured to use the Kubernetes API as its sole data source, run the following commands:
+
+> **Note**: The following manifests require Kubernetes 1.7.0 or later.
+{: .alert .alert-info}
+
+```shell
+kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/rbac-kdd.yaml
+kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calico-networking/1.7/calico.yaml
 ```
 
->[Click here to view the above yaml directly.](1.5/calico.yaml)
+>[Click here to view the above RBAC yaml directly.](../rbac-kdd.yaml)
+>
+>[Click here to view the above Calico yaml directly.](../kubernetes-datastore/calico-networking/1.7/calico.yaml)
 
-## Using calicoctl in a Kubeadm Cluster
+## Using calicoctl in a kubeadm cluster
 
-The simplest way to use calicoctl in Kubeadm is by running it as a pod.
+The simplest way to use calicoctl in kubeadm is by running it as a pod.
 See [using calicoctl with Kubernetes](../../../tutorials/using-calicoctl#b-running-calicoctl-as-a-kubernetes-pod) for more information.
 
 ## About
```

Comment:
This about section doesn't make sense any more given the changes in this document - it only applies to etcd but is scoped such that it looks like it applies to the kdd section as well.

I wonder if it's useful any more - I would probably vote to remove it entirely, perhaps splitting the information into other sections where it makes sense.

Reply:
It definitely feels more at home in the etcd section, the first two paragraphs seem helpfule, but I'm not sure how helpful it is to tell them how to check for the label, but it might be helpful to include what that label is. So I was thinking, drop the blurb about checking for the label, but specifying what that label is in the paragraph about how we pick where to run the etcd pod.

Reply:
I think at this point just remove the rest here - we don't need the checking for the label bit.